src/components/ListTask.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { useTasks } from '../hooks/useTasks';
+import { TaskItem } from './TaskItem';
+
+export const ListTask = () => {
+
+    const { tasks } = useTasks();
+
+    return (
+        <>
+            {tasks.map(task => (
+                <TaskItem key={task.title} task={task} />
+            ))}
+        </>
+  )
+}
```

Comment:
This component is being used only in one file. I think it enlarges boilerplate for now. It could be placed directly inside the `TodoContainer`.